### DISCUSSION TITLE
The website & preprint show different layouts - which is correct?

### DISCUSSION THREAD

**Discussion by NessDan** [binarybottle/engram-layout#27] [Q&A] [answered]
**Which is the correct layout?**

Looking at both the main website and the Github page, you can see ![this as the main layout for an ortholinear.](https://raw.githubusercontent.com/binarybottle/engram/master/assets/engram-ergo-squeezed-800px.png)

BUT on the [preprint,](https://www.preprints.org/manuscript/202103.0287/v1) the layout is ![incredibly different.](https://user-images.githubusercontent.com/304269/128544396-9b0a9262-427b-4346-a060-2ee7f1053cfc.png)

Which is the correct, most latest one? Also which one are all the stats based on?

**Comment by binarybottle** [comment, marked as answer]
The website is correct, not the preprint. The preprint manuscript is currently under review, and I am not permitted to revise it until I've heard back from the reviewers.

**Reply by NessDan** [reply]
Appreciate the response and thanks!

Is there a place where I can read up on why the preprint differs from the current version? Is the newer one just based on user feedback or some factor that was forgotten to be taken into account?

**Comment by binarybottle** [comment]
Feedback from the community inspired me to reevaluate the optimization criteria I was employing.  Please read the Jupyter notebook documentation to see the revised approach.

**Comment by binarybottle** [comment]
Please see https://github.com/binarybottle/engram-2021/discussions/86#discussion-10126586 for an announcement of the new Engram approach/layouts/repo!
